config/initializers/health_check.rb

```diff
@@ -0,0 +1,70 @@
+# frozen_string_literal: true
+
+HealthCheck.setup do |config|
+  # uri prefix (no leading slash)
+  config.uri = "health_check"
+
+  # Text output upon success
+  config.success = "success for following checks: ['database', 'migrations', 'email']"
+
+  # Timeout in seconds used when checking smtp server
+  config.smtp_timeout = 30.0
```

Comment:
Huh, that looks like a lot. We sure 30s is how long we want to let it run?

Reply:
Good point...

What is the email check doing exactly?

Reply:
I don't think we do. Notify takes care of it all.

Reply:
Removed